lib/shared/config-manager/src/index.ts

```diff
@@ -162,16 +277,24 @@ export class EnvironmentConfigManager {
             )
             return
         } else if (res?.status === 200 && projectConfig) {
+            const lastModifiedHeader = res?.headers.get('last-modified')
+            if (this.isLastModifiedHeaderOld(lastModifiedHeader)) {
+                this.logger.warn(
+                    'Skipping saving config, existing last modified date is newer.',
```

Comment:
is this important enough to be a warning? doesnt this just indicate things are working as expected?